src/Model/ASN.php

```diff
@@ -41,6 +41,7 @@
  */
 class ASN implements ModelInterface, ArrayAccess
 {
+    // use \Fingerprint\ServerAPI\Traits\WithRawResponse;
```

Comment:
Why is it commented? Same for all other models.

Reply:
Because raw response from http body is not required other than Response and EventResponse model. And since I am using generic model template, I need to enable this Trait only in specific models so I had to put comment

Reply:
I don't see a problem if this trait will be applied to other models too.

Reply:
I see. We can either apply to all models as @ilfa suggested, or think of an alternative solution.

Idea: let's refactor the `WithHttpInfo` methods, to return tuple with parsed response and raw response:
```php
  $serialized = ObjectSerializer::deserialize($responseBody, $returnType, []);
            $serialized->setRawResponse($responseBody);

            return [
                $serialized,
                $response,
            ];
```
that way, if someone wants to access raw response they can still do so by calling this method.
If they don't care about raw response, they can just use `getEvent`, `getVisits` etc.

Reply:
Done, we only have getVisits and getEvent methods for now and returning tuple with response instead of just serialized model